src/distdl/nn/conv_feature.py

```diff
@@ -1,13 +1,13 @@
 import numpy as np
 import torch
+import torch.nn.functional as F
 
+from distdl.backends.mpi.tensor_decomposition import compute_subtensor_shapes_unbalanced
```

Comment:
This should be done like it is in `transpose.py` to preserve back-end independence.

https://github.com/distdl/distdl/blob/master/src/distdl/nn/transpose.py#L188